.vscode/settings.json

```diff
@@ -2,5 +2,9 @@
   "files.associations": {
     "*.json": "jsonc"
   },
-  "graphql-config.load.rootDir": "packages/apps/graph"
+  "graphql-config.load.rootDir": "packages/apps/graph",
+  "cSpell.words": [
+    "Chainweaver",
+    "Kadena"
+  ]
```

Comment:
We have a (unwritten) convention to only add technical settings here that are required for certain extensions to work properly. 

Could you move this to your user setting? 

[User/Workspace settings](https://code.visualstudio.com/docs/getstarted/settings)
* [User settings](https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations)
* [Workspace settings](https://code.visualstudio.com/docs/getstarted/settings#_workspace-settingsjson-location)